src/care/util.h

```diff
@@ -15,6 +15,9 @@
 // Other library headers
 #if defined(__CUDACC__)
 #include "cuda.h"
+#endif
+
+#if defined(CARE_GPUCC)
 #if defined(CARE_DEBUG)
 #include "care/CUDAWatchpoint.h"
```

Comment:
Should this file be renamed GPUWatchpoint.h? Does it have both HIP and CUDA code in it?

Reply:
That's a good point. Yes, it is designed for both. Will do

Reply:
Done.